src/components/PopupWithForm.js

```diff
@@ -6,13 +6,16 @@ export default class PopupWithForm extends Popup {
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._inputItems = this._popupElement.querySelectorAll(".modal__input");
     this._handleFormSubmit = handleFormSubmit;
+    this._saveButton = this._popupForm.querySelector(".modal__button");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**